.github/workflows/bin-ci.yml

```
@@ -53,12 +53,12 @@ jobs:
     strategy:
       matrix:
         consul-version:
-        - 1.13.7
-        - 1.13.7+ent
         - 1.14.5
         - 1.14.5+ent
         - 1.15.1
         - 1.15.1+ent
```

If you want, you can bump the patch versions for 1.14 and 1.15 as well (per https://releases.hashicorp.com/consul/):

```suggestion
        - 1.14.8
        - 1.14.8+ent
        - 1.15.4
        - 1.15.4+ent
```